app/controllers/vendor_api/decisions_controller.rb

```diff
@@ -59,5 +55,20 @@ def invalid_metadata(_e)
                  }],
       }, status: 422
     end
+
+    def repond_to_decision(decision)
+      if decision.save
+        render json: { data: SingleApplicationPresenter.new(application_choice).as_json }
+      else
+        render_error_response(decision.errors)
+      end
+    end
+
+    # Takes a object with ActiveModel::Validations and render the `errors`
+    # as API response.
+    def render_error_response(errors)
```

Comment:
this is just for '422', should this be clear from the method name?
`render_422_error_response` or
`render_bad_request_error_response`

Reply:
Done